packages/ui-components/src/components/ComboBoxOption/ComboBoxOption.component.tsx

```diff
@@ -50,7 +49,14 @@ const truncateOptionStyles = `
   jn-whitespace-nowrap
 `
 
-export const ComboBoxOption = ({ children, disabled = false, value = "", label, className = "", ...props }) => {
+export const ComboBoxOption = ({
+  children,
+  disabled = false,
+  value = "",
+  label,
+  className = "",
+  ...props
+}: ComboBoxOptionProps) => {
   const comboBoxContext = useContext(ComboBoxContext)
```

Comment:
**(Suggested Improvement, Not Blocking)**

Any risk that ComboBoxContext would be `undefined` when we try to destructure it in later lines?
Should we add `if (!comboBoxContext) return null`?

Don't have to do it now, could add to [improvements](https://github.com/cloudoperators/juno/issues/570).

Reply:
I think that lack of context is invalid state and by rendering null we will just hide this fact.
I can add this to improvements, but I think it needs some more consideration.